### PR TITLE
Ignored return value SDL bug fix

### PR DESCRIPTION
**Pull request by orilevari**
**Description**: SDL bug fixes for warning C6031

### CODE REVIEW

**Review by ryanlai2 — DISMISSED** [review]

**Review by martinb35 — DISMISSED** [review]

File: winml/test/api/raw/winml_microsoft.h [hunk 1]
```diff
@@ -262,7 +262,10 @@ class WinMLLearningModel
 
     int32_t Initialize(const char* bytes, size_t size)
     {
-        RoInitialize(RO_INIT_TYPE::RO_INIT_SINGLETHREADED);
+      auto hr = RoInitialize(RO_INIT_TYPE::RO_INIT_SINGLETHREADED);
+      if (hr < 0 && hr != RPC_E_CHANGED_MODE) {
```
Comment (martinb35) [inline winml/test/api/raw/winml_microsoft.h:266]: can you add a comment about what RPC_E_CHANGED_MODE is and why it's a special case here?

**Comment by snnn** [comment]
Hi, any update?

**Review by ryanlai2 — APPROVED** [review]

**Review by tiagoshibata — APPROVED** [review]

**Comment by snnn** [comment]
The failures was because
 https://www.reddit.com/r/androiddev/comments/m5yeu4/httpsjcenterbintraycom_is_down/

**Pull request merged**